pkg/core/blockchain.go

```diff
@@ -876,10 +876,41 @@ func (bc *Blockchain) GetTransationResults(t *transaction.Transaction) []*transa
 	return results
 }
 
+// GetScriptHashesForVerifyingClaim returns all ScriptHashes of Claim transaction
```

Comment:
nit: "and checks Previous Index reference"

Reply:
Well, it has to do it because otherwise it'll panic with an out of bounds access, so there is not much to talk about.